packages/@angular/cli/tasks/eject.ts

```diff
@@ -287,7 +287,7 @@ class JsonWebpackSerializer {
         return v;
       } else if (typeof v == 'string') {
         if (v === path.join(this._root, 'node_modules')) {
-          return this._serializeRegExp(/\/node_modules\//);
+          return this._relativePath('process.cwd()', 'node_modules');
```

Comment:
We actually cannot rely on `process.cwd()` because the commands can be called from subfolders.

Can you replace this to `this._serializeRegExp(/(\\|\/)node_modules(\\|\/)/)` instead please? That way it supports both separators. This is also important for when the project is being ran in both Windows and OSX/Linux.